lib/Ed25519Signature2020.js

```diff
@@ -107,21 +111,24 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
   }
 
   async assertVerificationMethod({verificationMethod}) {
+    let contextUrl;
+    if(verificationMethod.type === 'Ed25519VerificationKey2018') {
+      contextUrl = SUITE_CONTEXT_URL_2018;
+    } else if(verificationMethod.type === 'Ed25519VerificationKey2020') {
+      contextUrl = SUITE_CONTEXT_URL;
+    } else {
```

Comment:
Make 2018 the fallback (presume 2020 will be used most of the time).

```suggestion
    if(verificationMethod.type === 'Ed25519VerificationKey2020') {
      contextUrl = SUITE_CONTEXT_URL;
    } else if(verificationMethod.type === 'Ed25519VerificationKey2018') {
      contextUrl = SUITE_CONTEXT_URL_2018;
    } else {
```

Reply:
addressed here 8fbde6a